rcl/test/CMakeLists.txt

```diff
@@ -61,7 +61,10 @@ function(test_target_function)
   )
 
   # TODO(wjwwood): remove this when the graph API works properly for more than OpenSplice.
-  if(rmw_implementation STREQUAL "rmw_opensplice_cpp")
+  if(
+    rmw_implementation STREQUAL "rmw_opensplice_cpp"
+    OR rmw_implementation STREQUAL "rmw_connext_cpp"
```

Comment:
Hmm, maybe this whole check should be removed now. I started these changes before FastRTPS got some upgrades. I'm testing that out right now.